tests/losses/test_contrastive_loss.py

```diff
@@ -18,21 +18,21 @@ def test_contrastive_loss() -> None:
         features=local_features.reshape(len(local_features), -1),
         positive_pairs=global_features.reshape(1, len(global_features), -1),
         negative_pairs=previous_local_features.reshape(1, len(previous_local_features), -1),
-    )
+    ).cpu()
```

Comment:
I'm not sure if you think this is better or not, but `.item()` automatically moves the value to cpu and produces a plain python float.

Reply:
yeah sure I can do that